src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -722,4 +724,11 @@ public static long onSleepFinished(ServerWorld world, long newTime, long minTime
         MinecraftForge.EVENT_BUS.post(event);
         return event.getNewTime();
     }
+
+    public static net.minecraftforge.event.entity.EntityEvent.Size getEntitySizeForge(Entity player, Pose pose, EntitySize size, float eyeHeight)
+    {
+        net.minecraftforge.event.entity.EntityEvent.Size evt = new net.minecraftforge.event.entity.EntityEvent.Size(player, pose, size, eyeHeight);
```

Comment:
You don't need to use fully qualified names here, just use `EntityEvent.Size` instead of `net.minecraftforge.event.entity.EntityEvent.Size`. Same in the line below